### PR TITLE
Added drive to objective feature.

### PR DESCRIPTION
**Pull request by aa15032261**
Also updated mod_blacklists.

### CODE REVIEW

**Comment by howdoiusekeyboard** [comment]
Would be cool if you could add auto_drive_to_objective

**Comment by aa15032261** [comment]
> Would be cool if you could add auto_drive_to_objective

Thanks for the suggestion. I’ll add that in.

**Comment by aa15032261** [comment]
> Would be cool if you could add auto_drive_to_objective

The feature is implemented in the latest commit.

File: BigBaseV2/src/util/blip.hpp [hunk 1]
```diff
@@ -1,22 +1,42 @@
...
 		Blip blip;
-		for (blip = HUD::GET_FIRST_BLIP_INFO_ID(sprite);
-			HUD::DOES_BLIP_EXIST(blip) &&
-			color != -1 && HUD::GET_BLIP_COLOUR(blip) != color;
+		for (
+			blip = HUD::GET_FIRST_BLIP_INFO_ID(sprite);
+			HUD::DOES_BLIP_EXIST(blip) && color != -1 && HUD::GET_BLIP_COLOUR(blip) != color;
 			blip = HUD::GET_NEXT_BLIP_INFO_ID(sprite)
-			) script::get_current()->yield();
+		);
```
Comment (Yimura) [inline BigBaseV2/src/util/blip.hpp:15]: Script yield gone, why?

I guess it's fine if the code can't deadlock.

File: BigBaseV2/src/backend/looped/vehicle/auto_drive.cpp [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+		) {
+			current_driving_flag = driving_style_flags[g->vehicle.auto_drive_style];
+			current_speed = g->vehicle.auto_drive_speed;
+			changing_driving_styles = true;
+		}
+
+		if (current_destination != AutoDriveDestination::STOPPED)
+		{
+			Vector3 last_waypoint = waypoint;
+			bool is_waypoint_exist = false;
```
Comment (Yimura) [inline BigBaseV2/src/backend/looped/vehicle/auto_drive.cpp]: `does_waypoint_exist` sounds better.

**Review by Yimura — CHANGES_REQUESTED** [review]

**Review by Yimura — APPROVED** [review]

**Comment by Yimura** [comment]
Seems to work, didn't really test in-depth.

**Pull request merged**